Update to luacrypto-0.3.1

romainmoreau opened the pull request:
I only tested to build with vs2010.

http://getmoai.com/forums/update-luacrypto-t924/

romainmoreau committed:
Update to luacrypto-0.3.1
3rdparty/luacrypto-0.2.0 removed
3rdparty/luacrypto-0.3.1 added
All matches in text files to "luacrypto-0.2.0" replaced by "luacrypto-0.3.1"

romainmoreau committed:
MOAIEnvironment.countryCode and MOAIEnvironment.languageCode support added for Android.

adamvillalobos commented:
If you could review http://getmoai.com/open-source-contributions.html and send us back the appropriate documents, I can merge this in soon.

romainmoreau committed:
MOAIEnvironment.screenDpi support added for Android.

romainmoreau commented:
I didn't want to add the last two commits to this pull request, but I can't find a way to remove them and add them in another pull request.
Anyway, these commits add the support of MOAIEnvironment.countryCode, MOAIEnvironment.languageCode and      MOAIEnvironment.screenDpi for Android.
I created a method AKUSetLocale because I know there is a way to listen to locale change but I didn't implemented it.
It would be great if all the other fields of MOAIEnvironment were supported for Android to.

romainmoreau committed:
MOAIEnvironment.cacheDirectory support added for Android.

romainmoreau committed:
USLog is now a dependency of MOAILogMgr only which is used instead to log.

romainmoreau committed:
MOAIShader_ShaderInfoLog_S logged with LOG_STATUS level.

romainmoreau committed:
Revert commits 0ecc4571d34e71c2f98a9a8edb5028eb217655fb, 767b79893355772a16ee78799042a0b220828e30 and fe383b850b0a481991e69b98202e7b68566af708.

romainmoreau committed:
Merge to Version-1.3-Build-160.

romainmoreau committed:
Update to luacrypto-0.3.1
3rdparty/luacrypto-0.2.0 removed
3rdparty/luacrypto-0.3.1 added
All matches in text files to "luacrypto-0.2.0" replaced by "luacrypto-0.3.1"

romainmoreau committed:
Update commit 50a2cc9e4739b503f0321fe9f69874fc0b2f424b from mkottman/luacrypto.

romainmoreau committed:
Fix GC actions witn parent : http://getmoai.com/forums/rare-crash-in-moaiaction-with-my-fix-t1469/

romainmoreau committed:
Ignore list

romainmoreau committed:
Visual Studio 2011

romainmoreau committed:
Ignore list

romainmoreau committed:
Allowing the frame buffer to be set at nil on a layer.

romainmoreau committed:
Adding getTimer and setTimer to MOAIApp for Android.

romainmoreau committed:
bare bones implementation of google billing v3. (f4162882d0898c2a23610bac3a3c39b587fa4be8)
+ BILLINGV3_RESPONSE_RESULT_UNUSED fix

makotok commented:
This pull request or still valid? 
I can not merge anymore. 

This pull request once closed.
